deps/SnoopCompile/snoopCompile.jl

```diff
@@ -0,0 +1,24 @@
+using SnoopCompile
+
+@snoopi_bot "Zygote" begin
```

Comment:
```suggestion
@snoopi_bot BotConfig("Zygote") begin
```
This is deprecated in the new version.